src/basic/hostname-util.h

```diff
@@ -60,3 +61,9 @@ static inline bool is_outbound_hostname(const char *hostname) {
         /* This tries to identify the valid syntaxes for the our synthetic "outbound" host. */
         return STRCASE_IN_SET(hostname, "_outbound", "_outbound.");
 }
+
+static inline int get_pretty_hostname(char **ret) {
+        assert(ret);
```

Comment:
parse_env_file does this anyway, and we don't dereference `ret` here, but just pass it along, hence I'd just drop this assert actually